cmd/spiffe-aws-assume-role/cli/cli_test.go

```diff
@@ -47,7 +46,7 @@ func TestErrorLogging(t *testing.T) {
 	err := RunWithDefaultContext(args)
 	require.Error(t, err)
 
-	bytes, err := ioutil.ReadFile(logFileName)
```

Comment:
sneaking that in here, eh?

Reply:
oh yes, lint won't pass without deprecating ioutil